archive/use-cases.html

```diff
@@ -62,6 +62,10 @@
     This specification defines an API to manage selected NFC use-cases from Web
     applications and pages, and to enable new use-cases based on NFC technology.
   </p>
+  <p class="advisement">
+    Low-level I/O operations with the ISO-DEP protocol and Host-based Card
```

Comment:
I don't know why we always say ISO-DEP - there are other low level protocols, but anyway that is a separate thing

Reply:
Shall I say "Low-level I/O operations such as with the ISO-DEP protocol" instead?

Reply:
I would also mention NFC-A/B, NFC-F or so - to make it obvious that more exist